test/babelish/test_csv2android.rb

```diff
@@ -11,6 +11,17 @@ def test_converting_csv_to_xml
     system("rm -rf ./values-en")
   end
 
+  def test_converting_csv_with_special_chars
+    csv_file = "test/data/android_special_chars.csv"
+    converter = Babelish::CSV2Android.new(csv_file, 'german' => "de")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.